### DISCUSSION TITLE
How am I supposed to submit a pull request?

### DISCUSSION THREAD

**Discussion by Francesco601** [MakeContributions/DSA#1178] [Q&A]
How am I supposed to submit a pull request or commit anything if my password is being rejected on the command line and the alternative was to creates a fine-grained identification thingie but that was not accepted either? If I click pull request on this page, it just says something is closed.

**Comment by sorohere** [comment]
Try using `SSH`, as I faced the same problem, but it worked well after switching. Also, make sure to clear your credentials cache.
